tests/unit/amo/components/TestAddon.js

```diff
@@ -1293,25 +1128,9 @@ describe(__filename, () => {
       const root = renderMoreAddons({ addon, addonsByAuthors });
 
       expect(root).not.toHaveClassName('.AddonDescription-more-addons--theme');
-      expect(root).toHaveProp('addons', addonsByAuthors.map((addonByAuthor) => (
-        createInternalAddon(addonByAuthor)
-      )));
-    });
-
-    it('indicates when other add-ons are themes', () => {
-      const addon = fakeTheme;
-      const addonsByAuthors = [
-        { ...fakeTheme },
-        { ...fakeTheme },
-        { ...fakeTheme },
-      ];
-
-      const root = renderMoreAddons({ addon, addonsByAuthors });
-
-      expect(root).toHaveClassName('.AddonDescription-more-addons--theme');
-      expect(root).toHaveProp('addons', addonsByAuthors.map((addonByAuthor) => (
-        createInternalAddon(addonByAuthor)
-      )));
+      expect(root).toHaveProp('authorNames',
```

Comment:
In this test, which is for checking the `MoreAddonsByAuthorsCard` component for an extension, I think we should also be checking that the component has the expected properties for `addonType` and `forAddonSlug`.

Reply:
Good catch!